scheduling_strawman.py

```diff
@@ -0,0 +1,77 @@
+"""
+The scheduler is a very useful tool. It requires the input of observing blocks
+and the time_range. Observing blocks consist of a target, a duration and constraints.
+"global" constraints can also be defined and can significantly speed up the scheduler.
+"""
+
+#a few packages are needed to set up the proper objects
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.time import Time
+import astroplan
+from astroplan.constraints import *
+from astroplan import Observer, FixedTarget
+from astroplan.scheduling import *
+
+#create the time frame during which you observing
+start_time = Time.now()+4*u.hour
+end_time = start_time+24*u.hour
+
+#define the observer
+mro = Observer.at_site('mro')
+
+#make the targets (this can also use FixedTarget(coords) if you make a
+#                   SkyCoord object coords)
+targets = [FixedTarget.from_name('vega'),
+           FixedTarget.from_name('Arcturus'),
+           FixedTarget.from_name('Altair'),
+           FixedTarget.from_name('Aldebaran'),
+           FixedTarget.from_name('Sirius'),
+           FixedTarget.from_name('Betelgeuse'),
+           FixedTarget.from_name('Rigel'),
+           FixedTarget.from_name('Castor')]
+target_group = [FixedTarget.from_name('Alcor'),
+                FixedTarget.from_name('M101')]
+#now ObservingBlocks and ScheduleBlocks need to be defined
+#priorities = {target.name : i for i, target in enumerate(targets)}#optional code
```

Comment:
I like this priority in the example, let's uncomment it and make it look like the rest of the important features.